modules/apps/commerce/commerce-product-test/src/testFunctional/tests/CPCommerceProductVisibility.testcase

```diff
@@ -38,13 +38,74 @@ definition {
 	}
 
 	@description = "This is a test for COMMERCE-6088. AddAccountGroupFilter"
-	@ignore = "true"
 	@priority = "3"
 	test AddAccountGroupFilter {
 		property portal.acceptance = "false";
 
-		// TODO COMMERCE-6088 AddAccountGroupFilter
+		CommerceNavigator.gotoPortlet(
+			category = "Product Management",
+			portlet = "Catalogs");
+
+		Click(locator1 = "Button#PLUS");
+
+		CommerceEntry.addNewCatalogsEntry(
+			catalogCurrency = "US Dollar",
+			catalogLanguage = "English (United States)",
+			catalogName = "Master");
+
+		AssertClickNoError(
+			locator1 = "Button#SUBMIT",
+			value1 = "Submit");
+
+		Refresh();
```

Comment:
Please remove the creation for the Master catalog because is already created when we do the reindex into the setUp (lines 15-20)